### DISCUSSION TITLE
Text highlighting

### DISCUSSION THREAD

**Discussion by mingyuecall** [marp-team/marp#389] [Q&A] [answered]
Could I ask for the code of highlighting text with background color in MARP?

==text== did not work... But it works on Typora.

Could you help me?

**Comment by yhatt** [comment, marked as answer]
The current Marp has no support `==highlight==` syntax because it is not supported by CommonMark syntax. But you can add syntax support by using markdown-it plugin in Marp CLI. See https://github.com/marp-team/marp-cli#example-markdown-it-mark.

Or you can use [`<mark>` HTML element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/mark) by enabling HTML: `<mark>highlight</mark>`

> :warning: First, don't use a word such as "Emergency" or "Important" to title (removed by moderator). Marp team would never triage by personal demand. A good question should not make pressure to community volunteers.

**Reply by mingyuecall** [reply]
I am so sorry for being rude in this way. I thought other users or your team might see and help.
Thank you team very much for the great work!
